Week2/todo_app/program.js

```diff
@@ -62,11 +123,20 @@ dbConnection.connect(function(err) {
     console.log('connected as id ' + dbConnection.threadId);
 
     const todoModel = new TodoModel(dbConnection);
-    todoModel.load(function(err, todoItems) {
-        if(err) {
-            console.log("error loading TODO items:", err);
+    // todoModel.load(function (err, todoItems) {
+    //     if (err) {
+    //         console.log("error loading TODO items:", err);
+    //     }
+
+    //     console.log("existing todo items:", todoItems);
+    // });
+
+    todoModel.delete(46, function (err, todoItems) {
```

Comment:
We'd like to make this app interactive - get input from the terminal and test all the functionalities you just made. Can you make this happen?